src/common/k8s-api/kube-object.store.ts

```diff
@@ -348,7 +348,7 @@ export abstract class KubeObjectStore<T extends KubeObject> extends ItemStore<T>
     const { signal } = abortController;
 
     const callback = (data: IKubeWatchEvent<T>, error: any) => {
-      if (!this.isLoaded || error?.type === "aborted") return;
+      if (!this.isLoaded || error instanceof DOMException || error?.type === "aborted") return;
```

Comment:
Probably just having `error instanceof DOMException` should be enough for aborted requests.
Or `error?.type === "aborted"` could come not only from DOM's `AbortController`?